tests_root/wdio/features/home.feature

```diff
@@ -0,0 +1,12 @@
+@Home_Screen
+Feature: Home Screen of Beep Web App
+  As a User
+  I want to navigate to the appropriate screen
+  So that I can check if my account name or password have been compromised
+
+  @S1 @manual
+  Scenario: Verification on fields on screen
```

Comment:
Removed data from the feature file. This should not contain any implementation details.